commands/configurations_test.go

```diff
@@ -226,9 +226,15 @@ func TestAddClusterConfigurationImpl(t *testing.T) {
 	restAPIMock := RestAPIMock{}
 
 	captured, err := capture.StandardOutput(func() {
-		os.Chdir("../")
+		err := os.Chdir("../")
```

Comment:
I think it will be nice to make function for it, WDYT?

Reply:
you mean a closure?

Reply:
actualy closure inside another closure :)